.github/workflows/model_uploader.yml

```diff
@@ -77,12 +77,12 @@ jobs:
       id: init_folders
       run: |
         model_id=${{ github.event.inputs.model_id }}
-        echo "model_folder=ml-models/${{github.event.inputs.model_source}}/${model_id}" >> $GITHUB_OUTPUT
         if [[ -n "${{ github.event.inputs.upload_prefix }}" ]]; then
           model_prefix="ml-models/${{ github.event.inputs.model_source }}/${{ github.event.inputs.upload_prefix }}"
         else
           model_prefix="ml-models/${{ github.event.inputs.model_source }}/${model_id%%/*}"
         fi
+        echo "model_folder=$model_prefix/${model_id##*/}" >> $GITHUB_OUTPUT
```

Comment:
"##" is used to match the longest {*/}. 
For example:
```bash
model_id="opensearch-project/opensearch-neural-sparse-encoding-v1/tree/main"
output="${model_id##*/}"
echo $output
```
will generate a main, because the ##*/ matched the "opensearch-project/opensearch-neural-sparse-encoding-v1/tree/" and deleted it.

Reply:
Please add a comment in the code, what does the `model_folder` look like for both cases (if and else)